pom.xml

```diff
@@ -1074,6 +1072,7 @@
                 <module>tensorflow-java</module>
                 <module>xstream</module>
                 <module>webrtc</module>
+				<module>libraries-2</module>
```

Comment:
let's add these modules in alphabetical order. Atleast that is what we are trying to maintain :)

Reply:
I will do

Reply:
Done